docs/videos/video-2-script-draft.txt

```diff
@@ -0,0 +1,19 @@
+How Event Bus Can Help You (And Veterans!)
+
+Now let's take a look at Event Bus in action, using a visual component to help see how the data flows through the system.
+
+Here we have a representation of our source data; in this case a medical claims processing event coming from a VA system. Our Veteran Mark has just had a medical procedure, and the event that the claim has been processed for procedure is about to be available. In the event data is relevant information that can help Mark if other VA systems know about it.
+
+This demo allows us to choose what data is included in the medical event; we'll select that the event requires a new prescription, and that the health update was important and Mark should get an SMS update about it. Now let's trigger that medical processing event!
+
+(Click Button)
+
+Now that we've triggered the event, it is made available to Event Bus, which picks it up immediately. Event Bus reads the event and sends pieces of the event data to specialized channels in real time.
```

Comment:
Is it really sending pieces? This makes it sound like Event Bus is parsing an event as it comes in and creating (for lack of a better term) "sub events" that go to various topics on the bus.

This is something that (I think) the bus could do, but is not how it is currently operating.

I think the key thing in this scenario is that the bus can put one event into multiple topics, so that systems listening to each of those topics can pick up the event. 

So maybe "Event Bus reads the event and sends pieces of the event data to specialized channels in real time." should be something like "Event Bus reads the event and sends it to one or more specialized channels in real time."

Reply:
The demo _is_ doing that, at least; when we were discussing how to build the demo, we specifically didn't want to advertise what we're doing with the DLA (filtering and deduplicating), so I was focusing more on what could be than what is, since we're not really doing what we idealize in our current reality.

From an architecture standpoint, if we're just passing on the same event, multiple consumers could listen to a single topic with those events instead of making a separate topic for each consumer and duplicating the event to them. Thinking of client-specific topics makes sense only if we plan to have each client have a number of differing events per topic tailored to that consumer; not sure if that was/is a design direction we had or want to go with instead? My vision was to have topics tailored to specific types of events that multiple clients could connect to if they want that type of event. More like a buffet (sub to all the specialized items you want) than a sit-down restaurant (waiter brings you all the different things they know you want to one place).

This is an interesting point though since I hadn't thought of having client specific topics. If that is the vision, i'd agree with changing this (although it'd mean the demo is not created to that vision).

Reply:
OK, if that is what the demo is doing, then I would revise my comment to say that we should be clear that this scenario is for demonstration purposes only and does not reflect any current use cases supported by the bus.

And yes, this is the general vision: "My vision was to have topics tailored to specific types of events that multiple clients could connect to if they want that type of event. More like a buffet (sub to all the specialized items you want) than a sit-down restaurant (waiter brings you all the different things they know you want to one place)."

Not so much client specific topics.

Reply:
I like it!